fastestimator/dataset/batch_dataset.py

```diff
@@ -287,3 +290,6 @@ def reset_index_maps(self, seed: Optional[int] = None) -> None:
                 else:
                     random.shuffle(mapping)
             self.index_maps.append([item for sublist in index_map for item in sublist])
+
+
+from fastestimator.dataset.extend_dataset import ExtendDataset
```

Comment:
what is this doing down here?

Reply:
It was creating an issue of circular import, which has this one solution.

Reply:
Solved this issue after introducing ExtendDataset in _init_.py file